UnityProject/Assets/Scripts/Systems/Lobby/CharacterManager.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+using UnityEngine;
+
+namespace Systems.Character
+{
+	/// <summary>Manage a player's characters. Intended for the local player client.</summary>
+	public class CharacterManager
+	{
+		public List<CharacterSheet> Characters { get; } = new();
+
+		private string OfflineStoragePath => $"{Application.persistentDataPath}characters.json";
```

Comment:
does this return the path correctly or is there a missing /

Reply:
This was how the path was formed before, it just uses string interpolation now.